src/index.js

```diff
@@ -25,12 +25,16 @@ function useVisibility(node, options = {}) {
   const observer = new IntersectionObserver(handleObserverUpdate, options);
 
   useEffect(() => {
-    if (node.current) {
-      observer.observe(node.current);
+    const element = node.current;
+
+    if (!element) {
```

Comment:
if there is no element we cant return a cleanup - because it would get called with no element and throw because of that